app/views/content/non-uk-teachers/teach-in-england-if-you-trained-overseas.md

```diff
@@ -33,204 +33,79 @@ expander:
-* [Teach in Scotland website](https://teachinscotland.scot/)
-* [Northern Ireland Department for Education website](https://www.education-ni.gov.uk/articles/initial-teacher-education-courses-northern-ireland)
+* a [competitive salary](/life-as-a-teacher/pay-and-benefits/teacher-pay)
+* support in your professional development
+* varied job roles
 
+You will need English [qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts) to teach in most schools in England. QTS is a professional teaching qualification that teachers in England gain at the end of their training. 
 
-If you do not want to come to England to train, you should consider [international qualified teacher status (iQTS)](/non-uk-teachers/international-qualified-teacher-status), which leads to the automatic award of English qualified teacher status. 
+You will also need a high standard of written and spoken English to teach in England. 
 
 $ukraine-moving$
 
-## What you will need to teach in England
-
-If you are already a qualified teacher, you do not need to retrain to teach in England. However, you will need:
-
-* to apply for English qualified teacher status (QTS), or be able to demonstrate strong non-UK qualifications and experience 
-* a high standard of written and spoken English
-* to pass criminal and professional safeguarding checks (these will be organised by your employer)
-* [a visa or immigration status allowing you to work in the UK](/non-uk-teachers/visas-for-non-uk-teachers)
-
-## Get an international relocation payment (IRP) worth up to $nonuk_internationalrelocationpayment_value$
-
-If you are interested in moving to England to teach languages or physics in an English secondary school, you could be eligible for up to $nonuk_internationalrelocationpayment_value$ from the UK government. 
-
-$irp$
-
-## English qualified teacher status (QTS)
+## If you do not have qualified teacher status
 
-Qualified teacher status (QTS) is the professional status teachers in England gain at the end of their teacher training. Schools use QTS as a measure of teaching skills and experience, so you will find it much easier to get a teaching job in England if you have QTS.
+If you do not have QTS, schools will want to see alternative evidence of your teaching and academic qualifications. 
 
-However, if you’re a qualified teacher from outside the UK, you can [work as a teacher in England for up to 4 years without QTS](https://www.gov.uk/government/publications/apply-for-qualified-teacher-status-qts-if-you-teach-outside-the-uk/routes-to-qualified-teacher-status-qts-for-teachers-and-those-with-teaching-experience-outside-the-uk#qts-exemption-for-teachers-from-outside-the-uk).
+You can get a [statement of comparability from the UK information centre for international qualifications and skills](https://www.enic.org.uk/Qualifications/SOC/Default.aspx). This will show how your qualifications compare to English ones. 
```

Comment:
Shouldn't we include `ENIC` as well as the full name? It is referred to as ENIC on other pages and also elsewhere in the journey (eg Apply)